src/Michał/Fsd.Michal.Cs.Ex2/FarmGenerator/FarmGenerator/GetUserData.cs

```diff
@@ -0,0 +1,112 @@
+﻿using FarmGenerator.Models;
+using System;
+using System.Collections.Generic;
+
+namespace FarmGenerator.Interfaces
+{
+    public class GetUserData
+    {
+        public static void GetInput(List<Animal> animalList)
+        {
+            CatInput(animalList);
+            DogInput(animalList);
+            CowInput(animalList);
+            SheepInput(animalList);
+        }
+
+        private static void DogInput(List<Animal> animalList)
```

Comment:
to bardzo słabe miejsce w aplikacji - gdy klient poprosi o 10 nowych gatunków zwierzaków, będziesz musiał powielać ten sam kod dla każdego datunku. Różnica jest tylko w 1 linijce. Koniecznie to zmień